compiler/codegen/symbol.cpp

```diff
@@ -55,6 +54,34 @@
 #include <sstream>
 #include <stdint.h>
 
+#ifdef HAVE_LLVM
+#include "llvm/Support/raw_os_ostream.h"
+#endif
+
+/******************************** | *********************************
+*                                                                   *
+*                                                                   *
+********************************* | ********************************/
+
+char llvmFuncDumpName[FUNC_NAME_MAX+1] = "";
+int llvmFuncOptDump = 0;
+
+#ifdef HAVE_LLVM
+void llvmFunctionDump(int optLevel, llvm::Function *llvmFunc, FnSymbol *chapelFunc) {
+  static llvm::Function *func = NULL; //Store function once we've found it
```

Comment:
This is really so that you can make the call in makeBinaryLLVM work nicely, right? Maybe put a comment to that effect?